pages/linux/zramctl.md

```diff
@@ -0,0 +1,23 @@
+# zramctl
+
+> Setup and control zram devices.
+
+- Check if zram is enabled:
+
+`lsmod | grep -i zram`
+
+- Enable zram with 2 devices:
```

Comment:
What does "device" mean here exactly ? Trying to understand zram a bit. IIUC, zram allows a sort of extended virtual memory to use RAM further and prevent paging to disk.

So how does "device" play in here. Is it a single zram "partition" / "disk" ? And if so, where is the size specified ?